Client/play/src/Status.jsx

```diff
@@ -24,22 +18,9 @@ module.exports = React.createClass({
     var currentPin = this.props.hunt.pins[currentPinIndex];
```

Comment:
Can we go over this together. It looks like there is some code in hear that no longer applies and needs to be stripped out.